models/author.rb

```diff
@@ -0,0 +1,16 @@
+class Author
+  attr_accessor :first_name, :last_name
+  attr_reader :id, :items
+
+  def initialize(first_name, last_name)
+    @first_name = first_name
+    @last_name = last_name
+    @id = Random.rand(1..1000)
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+    item.author = self
+  end
```

Comment:
- This function is wrong, as `Item class` does not have any `author` method. The inherited method name is `add_author`.
- Without implementing this change you will fail during **unit tests**
- Your `add_item` method can be implemented as below:
```
 def add_item(item)
    @items << item
    item.add_author(self)
  end
```

Reply:
@RitobrotoMukherjee thanks for the correction, I have solved